targets/TARGET_STM/sleep.c

```diff
@@ -97,19 +77,9 @@ static void ForceClockOutofDeepSleep(void)
     RCC_ClkInitStruct.AHBCLKDivider  = RCC_SYSCLK_DIV1;
     RCC_ClkInitStruct.APB1CLKDivider = RCC_HCLK_DIV1;
     if (HAL_RCC_ClockConfig(&RCC_ClkInitStruct, pFLatency) != HAL_OK) {
-        error("clock issue\r\n");
+		error("clock issue\r\n");
```

Comment:
8 spaces , not 4 here? Same issue is in the code below as well (copy-paste probably?)

Reply:
Recently changed my PC and have to reconfigure tools correctly - sorry about that ;-)